### PR TITLE
Add debug coredump instructions

### PR DESCRIPTION
**Pull request by KyleUltimate**
Add how to obtain the debug coredump.

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
Whether should this coredump also be recommended when reporting bugs

#### Is it ready for merging, or does it need work?
100% working

### CODE REVIEW

**Comment by vaxerski** [comment]
why do we need 2 methods? the first is simpler and in most cases enough

**Comment by KyleUltimate** [comment]
Yeah, most of the time the first method would be enough.
This pr is just a "just in case you need the debug dump* and you can just point the user to this page instead of teaching them how to do it

**Comment by vaxerski** [comment]
make it clearer then that it's "just in case"

Additionally, since most reporters are completely not into debugging, please make additional comments of continuing without paging and enabling debuginfod.

**Comment by KyleUltimate** [comment]
Do you mean this `set pagination off`?

**Comment by vaxerski** [comment]
no, gdb sometimes asks you to "press y to continue without paging"

File: docs/ISSUE_GUIDELINES.md [hunk 1]
```diff
@@ -57,3 +57,12 @@ coredumpctl info [PID]
...
 where `[PID]` is the PID you remembered.
 
+## Obtaining the debug Hyprland coredump
+In very rare cases, the normal coredump would not be enough.
+Only then, you could try obtaining the debug coredump.
+1. [Compile Hyprland with debug mode](http://wiki.hyprland.org/Contributing-and-Debugging/#build-in-debug-mode)
+1. Reproduce the crash in debug mode.
+2. `env DEBUGINFOD_URLS="https://debuginfod.archlinux.org/" coredumpctl debug [PID]`(see above)
+3. Wait until the `gdb` appears
+4. If gdb asks you `press y to continue without paging?` say no.
```
Comment (vaxerski) [inline docs/ISSUE_GUIDELINES.md:67]: what??? why?? Just continue without paging.

File: docs/ISSUE_GUIDELINES.md [hunk 2]
```diff
@@ -57,3 +57,12 @@ coredumpctl info [PID]
 ```
 where `[PID]` is the PID you remembered.
 
+## Obtaining the debug Hyprland coredump
+In very rare cases, the normal coredump would not be enough.
+Only then, you could try obtaining the debug coredump.
+1. [Compile Hyprland with debug mode](http://wiki.hyprland.org/Contributing-and-Debugging/#build-in-debug-mode)
```
Comment (vaxerski) [inline docs/ISSUE_GUIDELINES.md:63]: please mention that debug hyprland uses `hyprlandd.conf`

File: docs/ISSUE_GUIDELINES.md [hunk 3]
```diff
@@ -57,3 +57,12 @@ coredumpctl info [PID]
...
+1. Reproduce the crash in debug mode.
+2. `env DEBUGINFOD_URLS="https://debuginfod.archlinux.org/" coredumpctl debug [PID]`(see above)
+3. Wait until the `gdb` appears
```
Comment (vaxerski) [inline docs/ISSUE_GUIDELINES.md:66]: `(gdb)`

File: docs/ISSUE_GUIDELINES.md (hunk 1 above)
Comment (KyleUltimate) [reply docs/ISSUE_GUIDELINES.md:67]: Oh, wait I'm sorry
A brain fart I suppose

File: docs/ISSUE_GUIDELINES.md [hunk 4]
```diff
@@ -57,3 +57,14 @@ coredumpctl info [PID]
 ```
 where `[PID]` is the PID you remembered.
 
+## Obtaining the debug Hyprland coredump
+In very rare cases, the normal coredump would not be enough.
+Only then, you could try obtaining the debug coredump.
+1. [Compile Hyprland with debug mode](http://wiki.hyprland.org/Contributing-and-Debugging/#build-in-debug-mode)
+> Note: the config file will be in `$HOME/.config/hypr/hyprlandd.conf`
```
Comment (vaxerski) [inline docs/ISSUE_GUIDELINES.md:64]: *the config file used will be `hyprlandd.conf` instead of `hyprland.conf`*

**Pull request merged**